### ISSUE TITLE
Credential Attribute: Status List

### ISSUE THREAD

**Issue by jdsika** [ASCS-eV/credentials#9]
If you are not using a custom verifier then the credential must use a "credentialStatus" entry according to this [specification](https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/). Currently we hardcode the revocation registry in our application.

**Comment by jdsika** [comment]
We should use an identifier according to [EVES-007](https://github.com/ASCS-eV/EVES/blob/main/EVES/EVES-007/eves-007.md) that points to a revocation registry implemented like in this [paper](https://arxiv.org/pdf/2501.17089)

**Comment by flhps** [comment]
Just had a look at the example credentials in the PR and saw that they use a custom (?) set up for revocation with a `credentialSubject.revocationRegistryIndex` field (and registry in the issuer). Is that something coming from Gaia-X?

I would prefer adherence to the VC Data Model, which has a section on how revocation entries are structured. Adhering to that also has the advantage that we could easily add more status entries in the future, if needed. It encapsulates all required information for a status check in one json object. Here is the spec section: https://www.w3.org/TR/vc-data-model-2.0/#status

The compliant status section could look like:
```json
"credentialStatus": {
    "id": "did:web:did.identity.ascs.digital:revocation-registry#94567",
    "type": "CRSetEntry",
    "statusPurpose": "revocation"
  }
``` 
The last bit of id is specific revocation id; could also be separated with colon. Also, note that the used id is way too short to uphold security of CRSet from the paper above. Real ids would need to be much longer.

**Comment by jdsika** [comment]
No, I just did not look at the vc data model. Feel free to modify!

**Comment by jdsika** [comment]
Resolved in v2.0.0. All credentials include `credentialStatus` with `harbour:CRSetEntry` (Credential Revocation Set). Status is queryable via the revocation registry DID service endpoint. See EVES-008 §3.5.
